nrf_rpc/doc/transport.rst

```diff
@@ -0,0 +1,64 @@
+.. _transport:
+
+Transport
+#########
+
+Bottom layer of the nRF_RPC is a transport layer.
+It is OS-dependent.
+Its main goal is to transfer packets between two sites and to handle thread pool.
+The transport implementation can be selected by the configuration.
+
+Currently default transport is `OpenAMP <https://github.com/OpenAMP/open-amp/>`_ on `Zephyr <https://www.zephyrproject.org/>`_.
+
+For nRF_RPC top level API users it is not important to know details about the transport layer.
+This knowlage is important to implement a new transport or port to a different OS.
+
+The template header describing the nRF_RPC transport APIS is ``template/nrf_rpc_tr_tmpl.h``.
+Header file ``include/rp_trans.h`` is
```

Comment:
Something is missing here: is...

Reply:
Because of significant simplification of the transport layer, this file does not exists any more.